crates/matrix-sdk/src/event_cache/mod.rs

```diff
@@ -732,9 +736,9 @@ pub enum RoomEventCacheUpdate {
     },
 
     /// The room has new events.
-    Append {
+    SyncEvents {
         /// All the new events that have been added to the room's timeline.
-        events: Vec<SyncTimelineEvent>,
+        timeline: Vec<SyncTimelineEvent>,
```

Comment:
Please please please no, absolutely not. Timeline is already way too overloaded (as in "collection of events" or "the timeline API object from the UI crate), and it's much clearer to call this a collection of events, as one could imagine it's the full timeline from looking at the term here.

Reply:
I've renamed `Sync` to `ReceiveEventsFromSync` and the `timeline` field to `timeline_events`, it reads as `ReceiveEventsFromSync::timeline_events`, I believe there is no ambiguity here.